Inspicio/Controllers/AccountController.cs

```diff
@@ -122,6 +122,12 @@ public IActionResult Register(string returnUrl = null)
             if (ModelState.IsValid)
             {
                 var user = new ApplicationUser { UserName = model.Email, Email = model.Email };
```

Comment:
@Snappiestjack make sure you let everyone know, cos when they get that change the IDE might start shouting at them about some code style stuff :) 👍 on adding it though, makes it really easy to stick to the correct style.